addons/website_mail/static/src/js/follow.js

```diff
@@ -37,6 +37,14 @@ publicWidget.registry.follow = publicWidget.Widget.extend({
             self._toggleSubscription(true, data[0].email, $jsFollowToEnable);
             self._toggleSubscription(false, data[0].email, $jsFollowEls.not($jsFollowToEnable));
             $jsFollowEls.removeClass('d-none');
+            // TODO handle from xml in master
+            if (self.isUser) {
```

Comment:
Small note why we added input element explicitly here would be helpful to someone like groups="base.group_public" is added on input element hence in internal or portal use it can display only Subscribe/Unsubscribe butttons, this will add input if user is not public user.